extensions/amp-story/1.0/page-advancement.js

```diff
@@ -329,27 +333,55 @@ class ManualAdvancement extends AdvancementConfig {
 
     event.stopPropagation();
 
-    // TODO(newmuis): This will need to be flipped for RTL.
-    const elRect = this.element_./*OK*/getBoundingClientRect();
+    const pageRect = this.element_./*OK*/getBoundingClientRect();
 
     // Using `left` as a fallback since Safari returns a ClientRect in some
     // cases.
-    const offsetLeft = ('x' in elRect) ? elRect.x : elRect.left;
-    const offsetWidth = elRect.width;
+    const offsetLeft = ('x' in pageRect) ? pageRect.x : pageRect.left;
+    const offsetWidth = pageRect.width;
+    const bigArea = NEXT_SCREEN_AREA_RATIO * offsetWidth;
+    const smallArea = (1 - NEXT_SCREEN_AREA_RATIO) * offsetWidth;
+    const isRtl = this.storeService_.get(StateProperty.IS_RTL_STATE);
+
+    const sections = {
```

Comment:
Is there any way we can avoid creating a new instance of this object on every user interaction?

Maybe we could store the width as a ratio, and then do the operation in the `getTapDirection_` method.
ie:
````
left: {
  width: RTLState ? 0.75 : 0.25,
  direction: ...,
}
````